cockroachdb/src/jepsen/cockroach.clj

```diff
@@ -70,6 +70,10 @@
                   (auto/start! test node)
                   (Thread/sleep 5000)) ; Give it time to join
```

Comment:
I don't think this wait is necessary any more. It used to be necessary back in the day where it was unsafe to start a client while the node wasnt fully started yet. We fixed it.